docker/docker-compose.yml

```diff
@@ -26,6 +27,11 @@ services:
         environment:
             BUGSNAG_KEY: bbef84ab99c91665262f672ba6507a2d
             PG_PASSWORD: ${PW_POSTGRES}
+        healthcheck:
+            test: ["CMD", "curl", "--fail", "http://localhost/healthcheck"]
```

Comment:
two questions:
1) is this thing running in a container or in the host machine? (I suppose the former)
2) if in a container, should we specify the port as well? as it's not 80

Reply:
and the same questions for other PRs

Reply:
good question, i solved in all of them (which were btw superseded hehe)